src/SceneKit/SCNCompat.cs

```diff
@@ -111,4 +111,16 @@ static public void AddAnimation (this ISCNAnimatable self, SCNAnimation animatio
 		}
 	}
 #endif
+
+#if !XAMCORE_4_0
+	[Watch (3,0)]
+	public partial class SCNHitTestOptions {
+		[Obsolete ("Use 'SearchMode' instead.")]
+		public SCNHitTestSearchMode? OptionSearchMode {
+			get {
+				return (SCNHitTestSearchMode?) GetLongValue (SCNHitTest.SearchModeKey);
```

Comment:
minor: it would be better to `return SeachMode;` here - in case the generated code change (e.g. extra validation) over time.